src/main.rs

```diff
@@ -70,11 +69,23 @@ struct Opt {
     #[structopt(
         name = "level",
         long = "log",
-        help = "Use this flag to set the log level. Default is `warning`. Possible choices are [error, warning, info, debug, trace]"
+        help = "Use this flag to set the log level. Default is `warning`. Possible choices are [error, warning, info, debug, trace]."
     )]
     log: Option<log::Level>,
-    #[structopt(name = "speed", long = "speed", help = "Protocol speed in kHz")]
+    #[structopt(name = "speed", long = "speed", help = "The protocol speed in kHz.")]
     speed: Option<u32>,
+    #[structopt(
+        name = "restore-unwritten",
+        long = "restore-unwritten",
+        help = "Enable this flag to restore all bytes erased in the sector erase but not overwritten by any page. This option is only available with progressbars enabled."
```

Comment:
Why does this depend on progress bars?

Reply:
And if this is necessary, we should check for it and show an error if `--restore-unwritten` and `--disable-progressbars` are specified together.

Reply:
This is not true anymore. Will adjust :)